src/app/features/asset-list/stacks/stx-crypo-asset-item/stx-crypto-asset-item.stories.tsx

```diff
@@ -28,23 +28,43 @@ export default meta;
 
 type Story = StoryObj<typeof Component>;
 
-const symbol = 'STX';
+const symbolSTX = 'STX';
+const symbolUSD = 'USD';
```

Comment:
```suggestion
const symbolStx = 'STX';
const symbolUsd = 'USD';
```

https://www.notion.so/trustmachines/Naming-Camelcase-acronyms-119d74b694c78080974aee5e3bc18bc9?source=copy_link